### PR TITLE
Add promtail role for partition components.

### PR DESCRIPTION
**Pull request by Gerrit91**

### CODE REVIEW

File: control-plane/roles/logging/tasks/main.yaml [hunk 1]
```diff
@@ -12,19 +12,33 @@
...
+    apply: true
+  when: logging_ingress_loki_basic_auth
+
+- name: Deploy loki
   include_role:
     name: ansible-common/roles/helm-chart
   vars:
     helm_repo: "{{ logging_chart_repo }}"
-    helm_chart: loki-stack
+    helm_chart: loki
```
Comment (Gerrit91) [inline control-plane/roles/logging/tasks/main.yaml:34]: As we are changing to another chart, we need to adapt the release vector version: https://github.com/metal-stack/releases/blob/develop/release.yaml#L185-L187

File: control-plane/roles/logging/tasks/main.yaml [hunk 2]
```diff
@@ -12,19 +12,33 @@
...
-- name: Create monitoring ingress for Loki
-  k8s:
-    definition: "{{ lookup('template', 'loki_ingress.yaml') }}"
-    namespace: "{{ logging_namespace }}"
-    apply: true
+# - name: Create monitoring ingress for Loki
+#   k8s:
+#     definition: "{{ lookup('template', 'loki_ingress.yaml') }}"
+#     namespace: "{{ logging_namespace }}"
+#     apply: true
```
Comment (Gerrit91) [inline control-plane/roles/logging/tasks/main.yaml:44]: Can't remember if this is required or not, let's cleanup it up if not required and delete the template.

File: control-plane/roles/logging/tasks/main.yaml (hunk 1 above)
Comment (Gerrit91) [reply control-plane/roles/logging/tasks/main.yaml:34]: Let's also add an `ACTIONS_REQUIRED` section to the PR to indicate that the old helm installation has to be uninstalled manually before deployment.

File: partition/roles/promtail/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+# TODO: move to release vector
+promtail_image_name: grafana/promtail
+promtail_image_tag: 2.7.3
```
Comment (Gerrit91) [inline partition/roles/promtail/defaults/main.yml:4]: Would fit beneath `docker-images.third-party.partition` in the release vector.

File: partition/roles/promtail/templates/promtail.yaml.j2 [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  #     level:
+  #     component:
+  #     machineID:
+  # - metrics:
+  #     log_lines_total:
+  #       type: Counter
+  #       description: "total number of log lines"
+  #       source: component
+  #       config:
+  #         action: inc
```
Comment (Gerrit91) [inline partition/roles/promtail/templates/promtail.yaml.j2:41]: Can this be removed?

File: partition/roles/promtail/defaults/main.yml (hunk 3 above)
Comment (thanche1) [reply partition/roles/promtail/defaults/main.yml:4]: https://github.com/metal-stack/releases/pull/113

**Pull request closed without merging**